slime_environments/environments/SlimeEnvMultiAgent.py

```diff
@@ -325,18 +364,16 @@ def move(self):
             self.lay_pheromone(self.turtles[turtle]['pos'], self.lay_amount)
 
     # not using ".change_all" method form BooleanSpace
-    def last(self, current_agent):
+    def process_agent(self, current_agent):
         #self._evaporate()
         #self._diffuse()
 
         self.agent = current_agent
-        self.obs_dict[self.agent].change(0, self._compute_cluster(self.agent) >= self.cluster_threshold)
-        self.obs_dict[self.agent].change(1, self._check_chemical(self.agent))
-        cur_reward = self.reward_cluster_and_time_punish_time(self.agent)
-
-        return self.obs_dict[self.agent], cur_reward, False, {}
+        self.observations["agent_"+str(self.agent)] = np.array([self._compute_cluster(self.agent) >= self.cluster_threshold
+                                            ,self._check_chemical(self.agent)])
+        self.reward_cluster_and_time_punish_time(self.agent)
 
-    def lay_pheromone(self, pos: tuple[int, int], amount: int):

```

Comment:
are type hints removed by accident or due to petting zoo API compatibility=

Reply:
The type hints are resulting in syntax error in my pc, for some reason. Its not due to any compatibility reason.

Reply:
could be because of outdated python? I'm working with 3.11.3
(anyway I can take care of it, don't worry, I'm just telling in case you want type hints feature in your own code---which is nice)